test/e2e/components/vl-tooltip.js

```diff
@@ -49,15 +49,19 @@ class VlTooltip extends VlElement {
     }
 
     async _getTooltipId() {
-        const parentelement = await this.findElement(By.xpath('..'));
-        return parentelement.getAttribute('aria-describedby');
+        const parent = await this.parent();
+        return parent.getAttribute('aria-describedby');
     }
 
     async _getTooltipElement() {
-        const tooltipId  = await this._getTooltipId();
-        if (tooltipId) {
-            return new VlElement(this.driver, `#${tooltipId}`);
-        } else {
+        const tooltipId = await this._getTooltipId();
+        const parent = await this.parent();
+        const parentParent = await parent.parent();
+
+        try {
+            const tooltip = await parentParent.findElement(By.css(`#${tooltipId}`));
```

Comment:
zou je hier niet beter checken of de parent shadow dom heeft? nu is het een rare constructie die dan hopelijk in de beide gevallen (wel/niet shadow dom) goed komt. 
als er 2 cases zijn zie ik graag 2 cases staan ...

Reply:
Vroeger gingen we vanuit het document zoeken naar de tooltip, nu van de parent van de parent omdat de tooltip aangemaakt wordt als sibling van de parent. Een onderscheid lijkt me hier niet nodig.